Electron/src/RAM/Arm/Arm.tsx

```diff
@@ -18,6 +21,57 @@ const column: CSS.Properties = {
   marginRight: "5px",
 }
 
+function arm(): void {
+  let ArmWristBend = 0
+  let ArmWristTwist = 0
+  let ArmElbowTwist = 0
+  let ArmElbowBend = 0
+  let ArmBaseTwist = 0
+  let ArmBaseBend = 0
+  let Gripper = 0
+  let Nipper = 0
+
+  if ("WristBend" in controllerInputs && "WristTwist" in controllerInputs) {
+    ArmWristBend = controllerInputs.WristBend * controlMultipliers.Wrist
+    ArmWristTwist = controllerInputs.WristTwist * controlMultipliers.Wrist
+  }
+
+  if ("ElbowBend" in controllerInputs && "ElbowTwist" in controllerInputs) {
+    ArmElbowBend = controllerInputs.ElbowBend * controlMultipliers.Elbow
+    ArmElbowTwist = controllerInputs.ElbowTwist * controlMultipliers.Elbow
+  }
+
+  if ("BaseBendDirection" in controllerInputs && "BaseBendMagnitude" in controllerInputs) {
+    const direction = controllerInputs.BaseBendDirection === 1 ? -1 : 1
+    ArmBaseBend = direction * controllerInputs.BaseBendMagnitude * controlMultipliers.Base
+  }
+
+  if ("BaseTwistDirection" in controllerInputs && "BaseTwistMagnitude" in controllerInputs) {
+    const direction = controllerInputs.BaseTwistDirection === 1 ? -1 : 1
+    ArmBaseTwist = direction * controllerInputs.BaseTwistMagnitude * controlMultipliers.Base
+  }
+
+  if ("GripperOpen" in controllerInputs && "GripperClose" in controllerInputs) {
+    if (controllerInputs.GripperOpen === 1) {
+      Gripper = 1 * controlMultipliers.Gripper
+    } else if (controllerInputs.GripperClose === 1) {
+      Gripper = -1 * controlMultipliers.Gripper
+    } else {
+      Gripper = 0
+    }
+  }
+
+  if ("Nipper" in controllerInputs) {
+    Nipper = controllerInputs.Nipper
+  }
+
+  const armValues = [ArmWristBend, ArmWristTwist, ArmElbowTwist, ArmElbowBend, ArmBaseTwist, ArmBaseBend]
+
+  rovecomm.sendCommand("ArmVelocityControl", armValues)
+  rovecomm.sendCommand("GripperMove", Gripper)
```

Comment:
Are these being sent even if the arm control scheme was not selected? I see the arm function is run at a fixed interval